src/test/java/seedu/address/logic/commands/EditCompanyDescriptorTest.java

```diff
@@ -48,8 +47,8 @@ public void equals() {
         editedAmy = new EditCompanyDescriptorBuilder(DESC_GOOGLE).withEmail(VALID_EMAIL_TIKTOK).build();
         assertFalse(DESC_GOOGLE.equals(editedAmy));
 
-        // different tags -> returns false
-        editedAmy = new EditCompanyDescriptorBuilder(DESC_GOOGLE).withTags(VALID_TAG_HIGH).build();
+        // different priority -> returns false
+        editedAmy = new EditCompanyDescriptorBuilder(DESC_GOOGLE).withPriority("MEDIUM").build();
```

Comment:
Might have missed out changing `editedAmy` here, able to help change it to `editedGoogle` instead?

Reply:
done